app/models/manageiq/providers/vmware/infra_manager/event_catcher/runner.rb

```diff
@@ -101,13 +101,14 @@ def sub_type_and_name(event)
   private
 
   def worker_options
-    super.merge(
-      :ems => [
+    super.tap do |options|
+      options[:ems] = [
         @ems.attributes.merge(
           "endpoints"       => @ems.endpoints,
           "authentications" => @ems.authentications
         )
       ]
-    )
+      options.store_path(:settings, :ems, :ems_vmware, :blacklisted_event_names, @ems.blacklisted_event_names)
```

Comment:
The `blacklisted_event_names` in settings is `[]` by default and is merged with the class-method `.default_blacklisted_event_names`, this "fixes" the default event names missing in settings.

Reply:
Ultimately, I think this should go into _all_ of the worker options in the base class, but I'm ok with merging this as is right now.

Reply:
:+1: yeah agreed

Reply:
Since it ended up being impossible to add additional blacklisted_event_names via settings because of how this was implemented, I ended up moving all of the hard-coded `ManageIQ::Providers::Vmware::InfraManager.default_blacklisted_event_names` -> `config/settings.yml` and deleted this overridden method allowing the `BaseManager.default_blacklisted_event_names` to check the settings.  This also has the benefit of not requiring this hack to get the blacklisted event names to be available to non-rails workers